src/matchers/mock/toBeRequestedTimes.ts

```diff
@@ -0,0 +1,28 @@
+import { waitUntil, enhanceError } from '../../utils'
+import { runExpect } from '../../util/expectAdapter'
+
+function toBeRequestedTimesFn(received: WebdriverIO.Mock, times: number, options: ExpectWebdriverIO.CommandOptions = {}): any {
```

Comment:
I believe it might be helpful to support `{ gte, lte }`.
See https://github.com/webdriverio/expect-webdriverio/blob/a4f7a7ac03791929503cbeb1cc7409c6de8b43da/src/matchers/element/toHaveChildren.ts#L12 and https://github.com/webdriverio/expect-webdriverio/blob/53bfbba5ba4455d27042c8377ee130ae5093588d/src/utils.ts#L78

Reply:
see [comment](https://github.com/webdriverio/expect-webdriverio/pull/133#discussion_r464477457) from above.

Reply:
let's discuss it the thread above